src/app/pages/waiting-room-to-car/cat-adi-part2/components/vehicle-checks-question/vehicle-checks-question.cat-adi-part2.html

```diff
@@ -0,0 +1,44 @@
+<form>
+  <ion-row align-items-center [ngClass]="!isLastQuestion && 'mes-row-separator'"
+    class="mes-validated-row" id="vehicle-checks-question-row" [formGroup]="formGroup" nowrap>
+    <ion-col col-40>
```

Comment:
Anytime the HTML said ```col-x```, in Ionic 5 that is now ```size="x"```

Similarly, for any ```no-padding```, ```no-margin```, ```nowrap``` they need to live within the class now i.e.
```class="ion-no-padding ion-no-margin ion-no-wrap"```